### PR TITLE
Update node-asana

### PR DESCRIPTION
**Pull request by tkqubo**

### CODE REVIEW

**Comment by tkqubo** [comment]
I hope it's ok :)

File: lokijs/lokijs.d.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ interface Loki extends LokiEventEmitter {
      *   Example : var results = new loki().anonym(myDocArray).find({'age': {'$gt': 30} });
      * @param {Array} docs - document array to initialize the anonymous collection with
      * @param {Array} indexesArray - (Optional) array of property names to index
-     * @returns {Collection} New collection which you can query or chain
+     * @returns {SimpleResourceList} New collection which you can query or chain
```
Comment (vvakame) [inline lokijs/lokijs.d.ts]: Is this intentional change?

Comment (tkqubo) [inline lokijs/lokijs.d.ts]: Sorry they are not. I'll fix them!

Comment (vvakame) [inline lokijs/lokijs.d.ts]: please ping me again.

**Comment by tkqubo** [comment]
@vvakame unintentional changes fixed! :bow:

**Comment by vvakame** [comment]
:wink:

**Pull request merged**